src/monsters.cpp

```diff
@@ -220,9 +196,9 @@ std::list<Item*> MonsterType::createLootItem(const LootBlock& lootBlock)
 			tmpItem->setText(lootBlock.text);
 		}
 
-		itemList.push_back(tmpItem);
+		itemList.emplace_back(tmpItem);
```

Comment:
For built-in types (ints, pointers etc.) emplace_back works exactly the same as push_back. For complex types it behaves differently - it calls the move constructor if provided a copy or reference of an object, otherwise (eg. when given a const or const reference) it resorts to the copy constructor, if provided a list of arguments it actually in-place constructs the object and forwards them to the constructor.